packages/unmock-core/src/service/state/middleware.ts

```diff
@@ -62,7 +75,9 @@ export const spreadStateFromService = (
         // TODO do we want to throw for invalid types?
         const spread = {
           [key]:
-            isSchema(scm) && ajv.validate(scm, stateValue) ? stateValue : null,
+            isSchema(scm) && ajv.validate(scm, stateValue)
+              ? { ...scm, const: stateValue }
```

Comment:
`const` is not yet represented in `loas` types but it's officially part of JSON schema. This feels like the "correct" way to do this, rather than override the schema for a given key.